niimpy/test_screen.py

```diff
@@ -27,7 +27,7 @@ def test_screen_duration():
 
 @pytest.fixture
 def screen1():
-    return niimpy.read_csv(io.StringIO("""\
+    return niimpy.read_csv_string("""\
 time,screen_status
 0,1
```

Comment:
The test data has timestamps of unixtime, starting from 0.  I just thought of this for this project, but I think it makes a lot of sense.  The numbers are low, on a human scale of understanding.  And it's easy to know that 0 equals 1970-01-01 00:00 UTC or 02:00 EET.  So writing and looking at tests is really easy!